ui/welcome/src/commonMain/kotlin/social/androiddev/welcome/composables/WelcomeScreen.kt

```diff
@@ -0,0 +1,53 @@
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.material.Button
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import social.androiddev.common.theme.MastodonTheme
+
+@Composable
+fun WelcomeScreen(
+    modifier: Modifier = Modifier,
+) {
+    MastodonTheme {
```

Comment:
1) We should start using the `MastodonTheme` properly across all UI. This will open the door for someone else to tackle supporting Material You and building upon the theme as we focus on getting MVP functionality done...

2) Ignore this layout, it's just a dummy placeholder added to see if everything worked end-to-end. In a follow-up PR we can actually implement the full Welcome Screen with LogIn/SignUp flows.

Reply:
Let's move the `MastodonTheme` into the highest layer so the composable and screens are free to use in a separate app.

Reply:
Yup - take note of my comment. I just threw everything here to see it working end-to-end, including the theme. I'll be pulling that out and placing it at root in the next PR when I actually implement this screen.